src/change_stream.ts

```diff
@@ -1,5 +1,11 @@
 import Denque = require('denque');
-import { MongoError, AnyError, isResumableError, MongoDriverError } from './error';
+import {
+  MongoError,
+  AnyError,
+  isResumableError,
+  MongoDriverError,
+  MongoStreamClosedError
```

Comment:
As a general question to the team, do we want to conflate generic streams with change streams in our error reporting? Our change streams aren't really node streams in the sense that they don't extend from any of the base node streams. I could go either way, but my thought is that there could be some merit in differentiating "steam closed" vs "change stream closed" @nbbeeken @emadum Thoughts?

Reply:
Seems worthy of differentiating, I would imagine we'd only need use of MongoChangeStreamError, but there may be instances of Streams related errors, unless they stop something from continuing in our code I would say node stream errors should be bubbled up as is.

Reply:
The initial goal we had was to use `MongoStreamClosedError` between GridFSStream and ChangeStream to prevent creating two similar errors. If it's worth differentiating we could just split them down into `GridFSStreamClosed` and `ChangeStreamClosed`.